flake.nix

```diff
@@ -335,6 +335,30 @@
             }) output
           );
       };
+
+      htmlDocsSchema = {
+        version = 1;
+        doc = ''
+          The `htmlDocs` flake output defines packages providing static HTML output.
```

Comment:
It would be nice if we had a convention/requirement about what the entry point to the docs should be (e.g. there should be an `$out/index.html`) that we could document here (and maybe even enforce in the schema, though I don't think the meta-schema currently supports that). That way, tools like FlakeHub could provide a link to the docs automatically.

Currently this doesn't seem to be the case for Nixpkgs (it has entry points like `$out/share/doc/nixpkgs/index.html`).